src/main/java/com/google/api/codegen/gapic/MainGapicProviderFactory.java

```diff
@@ -421,6 +427,16 @@
           providers.add(messageProvider);
         }
       }
+      if (generatorConfig.enableTestGenerator() && id.equals(RUBY)) {
+        GapicProvider<?> testsProvider =
+            ViewModelGapicProvider.newBuilder()
+                .setModel(model)
+                .setApiConfig(apiConfig)
+                .setSnippetSetRunner(new CommonSnippetSetRunner(snippetContext))
+                .setModelToViewTransformer(new RubyTestsTransformer())
```

Comment:
Consider using 'RubyGapicSurfaceTestTransformer' instead of 'RubyTestsTransformer' in order to be consistent with other languages.